index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link  href="style.css" rel="stylesheet">
+   
+    <title>Zuhra Hashimi - Portfolio</title>
+</head>
+
+<body>
+    <div>
+           <nav class="topnav">
+                <a href="#home" class="logo">Zuhra Hashimi</a>
+                <div class="hamburger">
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                </div>
+            </nav>
+    </div>
+
+    <section class="main-content">
+        <div class="container">
+            
+            <h1 class="container headline">Hey There.<br>I'm Zuhra</h1>
+            <h2 class="container job-title">I am a web full stack developer </h2>
+            <p class="container job-description">I can help you build a product,<br>
+                feature or website.Look through some<br>
+                of my work and experience! If you like<br>
+                 what you see and have a project you need coded,<br>
+                  do not hestiate to contact me.</p>
+        </div>
+    
+        <div class="links">
+            <ul class="social_media">
+                <li>
+                    <a href="https://github.com/ZuhRa-HashiMi" id="git" target="blank"><img src="assets/github.svg" alt="github"></a>
+                </li>
+                <li>
+                    <a href="https://www.linkedin.com/in/zuhra-hashimi-601966214/" id="linkedin" target="blank"><img src="assets/Linkedin.svg" alt="linkedin"></a>
+                </li>
+             
+                <li>
+                    <a href="https://twitter.com/ZuhraHashimi4" id="twitter" target="blank"><img src="assets/twitter.svg" alt="twitter"></a>
+                </li>
+                <li>
+                    <a href="http://medium.com" id="medium" target="blank"><img src="assets/medium.svg" alt="Medium"></a>
+                </li>
+                <li>
+                    <a href="http://angellist.com" id="angel-list" target="blank"><img src="assets/angellist.svg" alt="angel-list"></a>
+                </li>
```

Comment:
- [ ] please sure to fix  the size of the icons as in the Figma design 
![image](https://user-images.githubusercontent.com/61790587/171284187-1e0dde15-23bd-4ab6-a685-56bff3ddbd09.png)